x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/metadata/add_pin_to_row.tsx

```diff
@@ -58,8 +61,22 @@ export const AddMetadataPinToRow = ({
     );
   }
 
+  // Custom table show on hover CSS, since this button is not technically in an action column
+  // Potential EUI TODO - multiple action columns and `align`ed actions are not currently supported
+  const showOnRowHoverCss = css`
+    opacity: 0;
+    ${euiCanAnimate} {
+      transition: opacity ${euiTheme.animation.normal} ${euiTheme.animation.resistance};
+    }
+
+    .euiTableRow:hover &,
+    &:focus-within {
+      opacity: 1;
+    }
+  `;
+
   return (
-    <span className="euiTableCellContent__hoverItem expandedItemActions__completelyHide">
+    <span className={showOnRowHoverCss}>
```

Comment:
@jennypavlova @elastic/infra-monitoring-ui I'd appreciate your help QAing this and confirming the hover UI/UX still works as expected. We've removed the above listed `className`s and the `showOnHover` API generally from columns, so this had to be converted to custom CSS instead.